docs/Debugging.md

```diff
@@ -0,0 +1,58 @@
+# Debugging Goblint
+
+## Building a Debuggable Goblint Executable
+
+To build a Goblint executable with debug information, run the following command within the `analyzer` directory.
+
+`make debug`
+
+## Debugging Goblint with VS Code
+
+To debug OCaml programs, you can use the command line interface of `ocamldebug` or make use of the Visual Studio Code
+integration provided by `hackwaly.ocamlearlybird`. In the following, we describe the steps necessary to set up this VS Code extension to
+debug Goblint.
+
+### Setting-up Earlybird
+
+Install the [`hackwaly.ocamlearlybird` extension](https://marketplace.visualstudio.com/items?itemName=hackwaly.ocamlearlybird) in your installation of Visual Studio Code.
+To be able to use this extension, you additionally need to install `ocamlearlybird` on the opam switch you use for Goblint. To do this, run the following command in the `analyzer` directory:
+
+`opam install earlybird`
```

Comment:
Maybe this should also be part of `make dev`?
https://github.com/goblint/analyzer/blob/11dcafdfd30fac8ba089840a633461ee08ddf3d1/make.sh#L98-L101